octobot_trading/modes/modes_factory.py

```diff
@@ -17,14 +17,13 @@
 import octobot_commons.logging as logging
 
 import octobot_trading.errors as errors
-import octobot_trading.modes as modes
 
 LOGGER_TAG = "TradingModeFactory"
 
 
 async def create_trading_modes(config: dict,
-                               exchange_manager: object,
-                               trading_mode_class: modes.AbstractTradingMode.__class__,
```

Comment:
?

Reply:
`object` is not accurate enough, it's triggering many IDE warnings so I think it's better not to type it at all.
importing `modes.AbstractTradingMode` is not possible anymore since `AbstractTradingMode` imports the factory

Reply:
Understood, thanks.